OrbitGl/Timer.h

```diff
@@ -5,32 +5,28 @@
 #ifndef ORBIT_GL_TIMER_H_
 #define ORBIT_GL_TIMER_H_
 
-#include <string>
-
-#include "GlUtils.h"
 #include "OrbitBase/Profiling.h"
 
 class Timer {
  public:
-  void Start() { start_ = MonotonicTimestampNs(); }
-
-  void Stop() { end_ = MonotonicTimestampNs(); }
+  explicit Timer() { Start(); }
 
+  void Start() { start_ = Now(); }
+  void Stop() { end_ = Now(); }
   void Reset() {
-    Stop();
-    Start();
+    start_ = Now();
+    end_ = 0;
   }
 
-  [[nodiscard]] double ElapsedMicros() const { return TicksToMicroseconds(start_, end_); }
-  [[nodiscard]] double ElapsedMillis() const { return ElapsedMicros() * 0.001; }
-  [[nodiscard]] double ElapsedSeconds() const { return ElapsedMicros() * 0.000001; }
-
-  [[nodiscard]] double QueryMillis() {
-    Stop();
-    return ElapsedMillis();
-  }
+  [[nodiscard]] double ElapsedNanos() const { return static_cast<double>(End() - start_); }
```

Comment:
Wouldn't it be better if this returned a `uint64_t`, as we always treat nanoseconds as `uint64_t` in the codebase and consider them the smallest time resolution?

Reply:
Good point, I changed it.